src/main/java/gyro/aws/ec2/PlacementGroupResource.java

```diff
@@ -0,0 +1,188 @@
+    }
+
+    public void setGroupName(String groupName) {
+        this.groupName = groupName;
+    }
+
+    /**
+     * One of three approaches towards managing the placement of instances on the underlying hardware (cluster | spread | partition).
+     * Defaults to the "cluster" strategy.
+     */
+    public PlacementStrategy getPlacementStrategy() {
+        return placementStrategy;
+    }
+
+    public void setPlacementStrategy(PlacementStrategy placementStrategy) {
+        this.placementStrategy = placementStrategy;
+    }
+
+    /**
+     * The number of partitions comprising the Placement Group.
+     * Ranges from 1 to 7. This value must be excluded unless using the "partition" strategy.
+     */
+    public Integer getPartitionCount() {
+        return partitionCount;
+    }
+
+    public void setPartitionCount(Integer partitionCount) {
+        this.partitionCount = partitionCount;
+    }
+
+    /**
+     * The ID of the Placement Group.
+     */
+    @Id
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    @Override
+    protected String getResourceId() {
+        return getId();
+    }
+
+    @Override
+    protected void doCreate(GyroUI ui, State state) {
+        Ec2Client client = createClient(Ec2Client.class);
+        CreatePlacementGroupRequest createRequest = CreatePlacementGroupRequest.builder().groupName(getGroupName()).strategy(getPlacementStrategy()).partitionCount(getPartitionCount()).build();
+        CreatePlacementGroupResponse createResponse = client.createPlacementGroup(createRequest);
+
+        DescribePlacementGroupsRequest describeRequest = DescribePlacementGroupsRequest.builder().groupNames(getGroupName()).build();
+        PlacementGroup group = null;
+        try {
+            DescribePlacementGroupsResponse describeResponse = client.describePlacementGroups(describeRequest);
+            if (!describeResponse.placementGroups().isEmpty()) {
+                group = describeResponse.placementGroups().get(0);
+            }
+
+        } catch (Ec2Exception ex) {
+            if (!ex.getLocalizedMessage().contains("unknown")) {
+                throw ex;
+            }
+        }
+
+        if (group != null) {
+            setId(group.groupId());
+        }
+
+    }
+
+    @Override
+    protected void doUpdate(GyroUI ui, State state, AwsResource config, Set<String> changedProperties) {
+
+    }
+
+    @Override
+    public boolean doRefresh() {
+        Ec2Client client = createClient(Ec2Client.class);
+        if (ObjectUtils.isBlank(getGroupName())) {
+            throw new GyroException("group name is missing, unable to load placement group.");
+        }
+
+        DescribePlacementGroupsRequest request = DescribePlacementGroupsRequest.builder().groupNames(getGroupName()).build();
+        PlacementGroup group = null;
+        try {
+            DescribePlacementGroupsResponse response = client.describePlacementGroups(request);
```

Comment:
Using lambda will help not using a separate variable

```suggestion
            DescribePlacementGroupsResponse response = client.describePlacementGroups(r -> r.groupNames(getGroupName()));
```